### PR TITLE
Add test for ext2fs and signing engine

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

File: samples/helloworld/ext2rootfs/Makefile [hunk 1]
```diff
@@ -28,7 +28,9 @@ package.pem:
 
 package: package.pem ext2rootfs
 	echo "Generating a signed package"
-	@myst package-sgx --roothash=roothash appdir package.pem ../config.json
```
Comment (mikbras) [inline samples/helloworld/ext2rootfs/Makefile:31]: Not sure why this PR has changes that are already merged into main.

File: tests/sign/Makefile [hunk 2]
```diff
@@ -0,0 +1,59 @@
+TOP=$(abspath ../..)
```
Comment (mikbras) [inline tests/sign/Makefile:1]: I wonder why this PR has changes already merged into main. The ``./tests/sign`` directory has already been merged.

**Review by mikbras — COMMENTED** [review]
Some changes shown in this PR are already merged into main. I am not sure what is going on with the git diffing engine.

File: samples/helloworld/ext2rootfs/Makefile (hunk 1 above)
Comment (salsal97) [reply samples/helloworld/ext2rootfs/Makefile:31]: I had cherry-picked your commit to include your changes, PR was merged at 10PM last night. Will rebase

File: tests/sign/Makefile (hunk 2 above)
Comment (salsal97) [reply tests/sign/Makefile:1]: Its because i cherry-picked your commit. It wasnt merged when this PR was created

**Review by vtikoo — APPROVED** [review]
LGTM

**Pull request merged**